esbuild.js

```diff
@@ -0,0 +1,16 @@
+const esbuild = require("esbuild");
+const { nodeExternalsPlugin } = require("esbuild-node-externals");
+
+// See the Bundling section of the readme for rationale/thoughts.
+
+esbuild.build({
+  entryPoints: ["src/index.ts"],
+  sourcemap: true,
+  outfile: "dist/index.mjs",
+  bundle: true,
+  format: "esm",
+  plugins: [nodeExternalsPlugin()],
```

Comment:
@bdow honestly I can't remember if this is really required, or if I was just mashing buttons; can you tell if we need it or not?

Reply:
Appears to build fine without it. Will push an update.